CI_YAML.md

```diff
@@ -42,7 +42,10 @@ targets:
 # presubmit: Whether to run this target on presubmit (defaults to true).
 # postsubmit: Whether to run this target on postsubmit (defaults to true).
 # run_if: List of path regexes that can trigger this target on presubmit.
-#         If none are passed, will always run in presubmit.
+#         If none are passed, it will evaluare run_if_not. If both are empty the target
+#         will always run in presubmit.
+# run_if_not: List of path regexes. The target will be run only if the files changed
```

Comment:
You should also mention that this applies only to presubmit targets. Same as run_if does.